okapi/services/caches/search/searching.inc.php

```diff
@@ -551,7 +551,29 @@ public function prepare_common_search_params()
                 }
             }
         }
-
+        
+        #
+        # powertrail_id
+        #
+        if (!is_null($tmp = $this->request->get_parameter('powertrail_id')))
+        {
+            if (!preg_match("/^(\d+(\|\d+)*)?$/", $tmp))
+                throw new InvalidParam('powertrail_id', "'$tmp'");
+            else {
+                if (Settings::get('OC_BRANCH') == 'oc.pl') {
+                    $extra_joins[] = 'inner join powerTrail_caches on powerTrail_caches.cacheId = caches.cache_id';
+                    #
+                    # Filter out caches from inactive powerTrails as well
+                    #
+                    $extra_joins[] = 'inner join PowerTrail on PowerTrail.id = powerTrail_caches.powerTrailId';
+                    $where_conds[] = 'PowerTrail.status = 1';
+                    $where_conds[] = "powerTrail_caches.powerTrailId in ( $tmp )";
```

Comment:
Tu jest chyba jakiś ewidentny bug. Wydaje mi się, że to się rozsypie gdy tylko ktoś poda więcej niż jeden ID...